wsd/ClientSession.hpp

```diff
@@ -82,6 +82,11 @@ class ClientSession final : public Session
 
     bool sendBinaryFrame(const char* buffer, int length) override
     {
+        if (isCloseFrame())
+        {
```

Comment:
vertical space is not a renewable resource =) this sort of thing bloats an otherwise readable header IMHO =)